File: sheet 9 S. Mahmoud and a Triangle/main.js
var n = +readline()
var segments = readline().split` `.map(x => +x)


function can_form_triangle() {
    segments.sort((a,b)=>a-b);
    for (var i = 0; i < n-2; i++) {
        if(segments[i]+segments[i+1]>segments[i+2]){
            return "YES"
        }
    }    
    return "NO"
}

print(can_form_triangle())

File: sheet 9 S. Mahmoud and a Triangle/main.py
def can_form_triangle(n:int, segments:list):
    segments.sort()
    for i in range(n - 2):
        if segments[i] + segments[i + 1] > segments[i + 2]:
            return "YES"
    return "NO"

n = int(input())
segments = list(map(int, input().split()))

result = can_form_triangle(n, segments)
print(result)
